R/tidyverse.R

```diff
@@ -19,18 +19,18 @@
 #'   the currently installed version.
 #'
 #' * `use_tidy_contributing()`: creates tidyverse contributing guidelines from
-#'    tidy_contributing template.
+#'   tidy_contributing template.
 #'
 #' * `use_tidy_issue_template()`: creates a standard tidyverse issue template.
 #'
 #' * `use_tidy_support()`: creates support resources document for GitHub repo
-#'    using SUPPORT.md.
+#'   using SUPPORT.md.
 #'
 #' * `use_tidy_coc()`: creates code of conduct same as use_code_of_conduct
```

Comment:
"use_code_of_conduct" still needs the backtick-and-parens treatment 😅

I'm not giving up this time!

Reply:
_OH_! Ok, ok! I see what you mean. I actually _did_ have that at one point, which I think just arbitrarily coincided with my getting an error when I ran R CMD Check, and I just assumed they were related!